packages/eas-cli/src/project/android/applicationId.ts

```diff
@@ -159,10 +159,10 @@ export function warnIfAndroidPackageDefinedInAppConfigForBareWorkflowProject(
 ): void {
   if (AndroidConfig.Package.getPackage(exp) && !warnPrinted) {
     Log.warn(
-      `Specifying "android.package" in ${getProjectConfigDescription(
+      `Specified value for "android.package" in ${getProjectConfigDescription(
         projectDir
-      )} is deprecated for bare workflow projects.\n` +
-        'EAS Build depends only on the value in the native code. Please remove the deprecated configuration.'
+      )} is ignored if native project folders are detected.\n` +
```

Comment:
```suggestion
      )} is ignored because an ${chalk.bold('android')} directory was detected in the project, indicating that this is a bare workflow project.\n` +
```

a bit wordy but
- lets people know what triggered this detection (although we could possibly be more precise - detected dir and wasn't gitignored)
- factors in @wkozyra95's comment about bare workflow

Reply:
See my comment above. I specifically moved away from saying 'bare workflow'. I do like the more specific folder naming.